src/pages/Location/SoilCondition/SoilDrainage/RenderDrainageClasses.js

```diff
@@ -21,7 +21,7 @@ const RenderDrainageClasses = ({
   const historyStateRedux = useSelector((stateRedux) => stateRedux.userData.historyState);
 
   const [previousDrainage, setPreviousDrainage] = useState(-1);
-  const [updateTilingCheck, setUpdateTilingCheck] = useState(true);
+  const [updateTilingCheck, setUpdateTilingCheck] = useState(false);
```

Comment:
this needs to be put in redux. selecting tiling as yes, then clicking next, and then back no longer shows tiling as yes

Reply:
this will be a new issue